Lib/test/support/_hypothesis_stubs/_helpers.py

```diff
@@ -0,0 +1,34 @@
+# Stub out only the subset of the interface that we actually use in our tests.
+class StubClass:
+    def __init__(self, *args, **kwargs):
+        super().__init__()
```

Comment:
Presumably nothing, but I believe this is relevant when there is some more complicated MRO.

TBH I can never keep the details straight and I can't remember if I did this because it was necessary or out of habit. Looking at it now, it seems like this is the base of a linear hierarchy, so right now `super().__init__()` will always call `object.__init__()`, but I'm not sure it's *hurting* anything...

Reply:
It raises questions in a reviewers/readers mind, such as "Did you forget to subclass from somthing else?"

Reply:
Done.